src/Mount.cpp

```diff
@@ -3740,3 +3884,33 @@ void Mount::testUART_vactual(TMC2209Stepper *driver, int _speed, int _duration)
 }
     #endif
 #endif
+
+/////////////////////////////////
+//
+// checkRALimit
+//
+/////////////////////////////////
+void Mount::checkRALimit()
+{
+    // Check tracking limits every 5 seconds
+    if (millis() - _lastTRKCheck > 5000)
+    {
+        float trackedHours       = (_stepperTRK->currentPosition() / _trackingSpeed) / 3600.0F;  // steps / steps/s / 3600 = hours
+        float homeRA             = _zeroPosRA.getTotalHours() + trackedHours;
+        float const RALimit      = RA_TRACKING_LIMIT;
+        float homeCurrentDeltaRA = homeRA - currentRA().getTotalHours();
+
+        LOGV2(DEBUG_MOUNT_VERBOSE, F("[MOUNT]: checkRALimit: homeRAdelta: %f"), homeCurrentDeltaRA);
+        while (homeCurrentDeltaRA > 12)
+            homeCurrentDeltaRA = homeCurrentDeltaRA - 24;
```

Comment:
`homeCurrentDeltaRA -= 24.0f` helps to show intent here by making the statement a bit less verbose